src/components/pages/my-bookings/MyBookingsLists/BookingsList/BookingItem/BookingItem.jsx

```diff
@@ -79,6 +79,7 @@ BookingItem.defaultProps = {
 BookingItem.propTypes = {
   booking: PropTypes.shape({
     id: PropTypes.string,
+    qrCode: PropTypes.string,
```

Comment:
```suggestion
    isQrCode: PropTypes.bool,
```
Avec une valeur par défaut.

Reply:
C'est la valeur du QR Code et non un booléen